cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-global-transformations.php

```diff
@@ -402,11 +402,15 @@ private function init_taxonomy_manager( $post ) {
 			}
 		}
 		$out[] = '</ul>';
+		
 		// Get apply Type.
-		$type  = get_post_meta( $post->ID, self::META_APPLY_KEY . '_terms', true );
-		$out[] = '<label class="cld-tax-order-list-type"><input ' . checked( 'overwrite', $type, false ) . ' type="checkbox" value="overwrite" name="cld_apply_type" />' . __( 'Overwrite', 'cloudinary' ) . '</label>';
+		if ( ! empty( $terms ) ) {
+			$type  = get_post_meta($post->ID, self::META_APPLY_KEY . '_terms', true);
```

Comment:
@dugajean 

It seems that we are removing the spaces inside the parenthesis.

Reply:
```suggestion
			$type  = get_post_meta( $post->ID, self::META_APPLY_KEY . '_terms', true );
```